assets/stylesheets/mobile/chat-drawer.scss

```diff
@@ -0,0 +1,63 @@
+.full-page-chat {
+  .channels-list {
+    background: var(--primary-very-low);
+
+    .chat-channel-row {
+      padding: 0.5rem 1rem;
+      &:not(:last-child) {
+        &::after {
+          content: "";
+          display: block;
+          width: 90%;
+          padding-top: 20px;
+          border-bottom: 1px solid var(--primary-low);
+          margin: auto;
+          position: absolute;
+          bottom: 0;
+          left: 0;
+          right: 0;
+        }
+      }
+    }
+
+    .chat-channel-divider {
+      padding: 1rem 2rem;
+
+      .channel-title {
+        color: var(--quaternary);
+      }
+    }
+
+    .channels-list-container {
+      background: var(--header_background);
+    }
+  }
+
+  .btn-floating.new-dm {
+    position: absolute;
+    background: var(--tertiary);
+    bottom: 1vh;
```

Comment:
this 1vh is a little bit unexpected any reason we don't go for fixed spacing?

Reply:
oops that was unintentional, that was meant to `1rem`. I'll update that thanks!